From f32dc27e15fcd7f4a00e7d832950d1f565ff7aa3 Mon Sep 17 00:00:00 2001
From: gv1222 <gvelji@gmail.com>
Date: Mon, 23 Feb 2015 16:53:43 -0500
Subject: [PATCH] Center dim button

---
 www/css/cytube.css | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/www/css/cytube.css b/www/css/cytube.css
index 8d35233..802d410 100644
--- a/www/css/cytube.css
+++ b/www/css/cytube.css
@@ -611,7 +611,7 @@ table td {
 }
 
 #dim {
-    margin-top: 5px;
-    margin-right: 5px;
+    margin-top: 7px;
+    margin-right: 6px;
     float: right;
 }
\ No newline at end of file
-- 
1.8.3.msysgit.0

From 7a51f625bbe9b0cfe4d7bd517703ae66c402b119 Mon Sep 17 00:00:00 2001
From: gv1222 <gvelji@gmail.com>
Date: Sun, 22 Feb 2015 16:56:27 -0500
Subject: [PATCH] Complete dim button

---
 templates/channel.jade | 2 +-
 www/css/cytube.css     | 2 +-
 www/js/ui.js           | 4 ++--
 3 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/templates/channel.jade b/templates/channel.jade
index 6430168..0fa7a5c 100644
--- a/templates/channel.jade
+++ b/templates/channel.jade
@@ -6,7 +6,6 @@ html(lang="en")
     link(href="//code.jquery.com/ui/1.10.3/themes/smoothness/jquery-ui.css", rel="stylesheet")
   body
     #wrap
-    #bgoverlay
       nav.navbar.navbar-inverse.navbar-fixed-top(role="navigation")
         include nav
         mixin navheader()
@@ -214,6 +213,7 @@ html(lang="en")
           .modal-footer
             button.btn.btn-default(type="button", data-dismiss="modal") Close
     #pmbar
+    #bgoverlay
     include footer
     mixin footer()
     script(src=sioSource)
diff --git a/www/css/cytube.css b/www/css/cytube.css
index 00a676a..8d35233 100644
--- a/www/css/cytube.css
+++ b/www/css/cytube.css
@@ -597,7 +597,7 @@ table td {
     bottom: 0;
     left: 0;
     right: 0;
-    background-color: #1c1e22;
+    background-color: #090A0C;
     opacity: .9;
     display: block;
     z-index: 1001;
diff --git a/www/js/ui.js b/www/js/ui.js
index 5839c47..89c7da4 100644
--- a/www/js/ui.js
+++ b/www/js/ui.js
@@ -22,8 +22,8 @@ $("#togglemotd").click(function () {
     }
 });
 
-$("dim").click(function () {
-    $("bgoverlay").toggle();
+$("#dim").click(function () {
+    $("#bgoverlay").toggle();
 });
 
 /* chatbox */
-- 
1.8.3.msysgit.0

From 4286b18f30212075288e0a0852c6db24af8bcd38 Mon Sep 17 00:00:00 2001
From: gv1222 <gvelji@gmail.com>
Date: Sun, 22 Feb 2015 16:29:39 -0500
Subject: [PATCH] Implement dim button

---
 templates/channel.jade |  2 ++
 www/css/cytube.css     | 25 +++++++++++++++++++++++++
 www/js/ui.js           |  4 ++++
 3 files changed, 31 insertions(+)

diff --git a/templates/channel.jade b/templates/channel.jade
index c0f1aa9..6430168 100644
--- a/templates/channel.jade
+++ b/templates/channel.jade
@@ -6,6 +6,7 @@ html(lang="en")
     link(href="//code.jquery.com/ui/1.10.3/themes/smoothness/jquery-ui.css", rel="stylesheet")
   body
     #wrap
+    #bgoverlay
       nav.navbar.navbar-inverse.navbar-fixed-top(role="navigation")
         include nav
         mixin navheader()
@@ -49,6 +50,7 @@ html(lang="en")
                 span.input-group-addon Guest login
                 input#guestname.form-control(type="text", placeholder="Name")
             #videowrap.col-lg-7.col-md-7
+              span#dim.glyphicon.glyphicon-adjust.pointer
               p#currenttitle Nothing Playing
               .embed-responsive.embed-responsive-16by9
                 #ytapiplayer.embed-responsive-item
diff --git a/www/css/cytube.css b/www/css/cytube.css
index 153f26d..00a676a 100644
--- a/www/css/cytube.css
+++ b/www/css/cytube.css
@@ -590,3 +590,28 @@ table td {
     border: 1px solid;
     border-top-width: 0;
 }
+
+#bgoverlay {
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    background-color: #1c1e22;
+    opacity: .9;
+    display: block;
+    z-index: 1001;
+    display: none;
+}
+
+#videowrap, #chatwrap, #dim {
+    opacity: 1 !important;
+    z-index: 1002 !important;
+    position: relative !important;
+}
+
+#dim {
+    margin-top: 5px;
+    margin-right: 5px;
+    float: right;
+}
\ No newline at end of file
diff --git a/www/js/ui.js b/www/js/ui.js
index 66a3484..5839c47 100644
--- a/www/js/ui.js
+++ b/www/js/ui.js
@@ -22,6 +22,10 @@ $("#togglemotd").click(function () {
     }
 });
 
+$("dim").click(function () {
+    $("bgoverlay").toggle();
+});
+
 /* chatbox */
 
 $("#modflair").click(function () {
-- 
1.8.3.msysgit.0

